Sources/ProjectDescription/FileList.swift

```diff
@@ -2,28 +2,47 @@ import Foundation
 
 public struct FileList: Codable, Equatable {
     /// List glob patterns.
-    public let globs: [Path]
+    public let globs: [HeaderFileGlob]
```

Comment:
I find it confusing that a `FileList` is now coupled to headers through `[HeaderFileGlob]`. I don't see the implementation of `HeaderFileGlob` being headers-specific so what about renaming that one to something like `FileListGlob`?

Reply:
FileList used only in headers, seems like it's better in the next major version to rename it by adding Header prefix.
But if you're not agree - I will rename it to FileListGlob - no problem

Reply:
Renamed